CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - `stream_register`: Fix `DATA_WIDTH` of instantiated FIFO.
 - `stream_xbar`: Add missing argument in assertion error string.
+- Lint style fixes
 
```

Comment:
I suggest you add
- deprecated `sram`
- added `stream_fork_seq`

to the Changelog.

Reply:
Thanks a lot, I absolutely agree. I didn't want to introduce `stream_fork_seq`. Thanks for catching that. Absolutely agree regarding the `sram`.